test/net/sourceforge/kolmafia/maximizer/MaximizerRegressionTest.java

```diff
@@ -132,30 +132,27 @@ public void equipEmptyCrownNoSlot() {
   }
 
   @Test
-  public void keepFilledBjornEquippedWithBonus() {
-    canUse("vampyric cloake");
+  public void keepBjornEquippedWithBonus() {
+    // Provide a helpful alternative to the bjorn.
+    canUse("makeshift cape");
     equip(EquipmentManager.CONTAINER, "Buddy Bjorn");
-    KoLCharacter.setBjorned(new FamiliarData(FamiliarPool.DICE));
 
-    assertTrue(maximize("adventures, -buddy-bjorn, +25 bonus Buddy Bjorn"));
-    // Fuzzy Dice adds +10 to all stats. Make sure it's still on.
-    assertEquals(10, modFor("Muscle"), 0.01);
+    assertTrue(maximize("-item, -buddy-bjorn, +25 bonus Buddy Bjorn"));
```

Comment:
you're maximizing `-item`, you probably want `item` for a meaningful test?

Reply:
Yep! I'm switching this back to adventures to be consistent with the other test. I dont remember when I changed that. Good catch!